### ISSUE TITLE
cannot add content-type to CFT

### ISSUE THREAD

**Issue by smichea** [meveo-org/meveo#451]
I create a binary CFT with content-type `application/zip` then save the entity.
I edit the CFT and add the content-type `application/x-zip-compressed`
I get the exception in the log (and nothing in the GUI) :
```
2021-11-28 00:33:23,628 SEVERE [javax.enterprise.resource.webcontainer.jsf.context] (default task-2189) java.lang.UnsupportedOperationException
	at java.base/java.util.AbstractList.add(AbstractList.java:153)
	at java.base/java.util.AbstractList.add(AbstractList.java:111)
	at deployment.meveo.war//org.primefaces.component.collector.Collector.processAction(Collector.java:71)
	at javax.faces.api@3.0.0.Final//javax.faces.event.ActionEvent.processListener(ActionEvent.java:72)
```

**Comment by czetsuya** [comment]
@smichea seems like Primefaces issue. This feature is available on [v8](https://www.primefaces.org/showcase-v8/ui/misc/collector.xhtml) we are using v7.

**Comment by tonysviews** [comment]
Issue will be addressed in #452
